### PR TITLE
Ingesting adapt

### LINKED ISSUE #37: Should work with ADAPT maps
This is a feature request to add ADAPT maps as an option to fluxpipe.

### PR DESCRIPTION
**Pull request by GillySpace27**

### CODE REVIEW

File: pdl/t/00-test.t
Comment (lowderchris) [inline pdl/t/00-test.t:1]: Leave these changes out - they'll be merged in another pull request.

File: pdl/PDL/map_fluxon_flow_parallel_master.pdl
Comment (lowderchris) [inline pdl/PDL/map_fluxon_flow_parallel_master.pdl:1]: Delete commented out lines.

File: flux-pipe/relax_pfss_world.pdl [hunk 1]
```diff
@@ -220,6 +220,9 @@ sub relax_pfss_world {
         # my $short_world_out_path = shorten_path($world_out_dir);
 
         # # print "\n\tWorld saved to $short_world_out_path";
+    # print $this_world_orig;
```
Comment (lowderchris) [inline flux-pipe/relax_pfss_world.pdl]: Commented out lines as an addition here?

Comment (GillySpace27) [reply flux-pipe/relax_pfss_world.pdl]: Yeah, I guess I could switch it to "if False: print"? Or do you want all commented lines to go away?

File: flux-pipe/plot_wind_map_detailed.py [hunk 2]
```diff
@@ -278,19 +278,21 @@ def plot_wind_map_detailed(args):
...
+    dat_dir =   configs.get("data_dir")
+    nwant =     configs.get("nwant")
+    CR =        configs.get("cr")
+    dat_file =  configs.get("file", f'{dat_dir}/batches/{batch}/cr{CR}/wind/cr{CR}_f{nwant}_radial_wind.dat')
+    reduce_amt = configs.get("mag_reduce")
 
     # Load the magnetogram parameters
-    (hdr, cr, fname, adapt, doplot, reduce) = load_magnetogram_params(dat_dir)
-    CR = args.cr or cr
+    # (hdr, cr, fname, adapt, doplot, reduce) = load_magnetogram_params(dat_dir)
```
Comment (lowderchris) [inline flux-pipe/plot_wind_map_detailed.py:292]: Are these commented out lines required any longer?

File: flux-pipe/relax_pfss_world.pdl (hunk 1 above)
Comment (lowderchris) [reply flux-pipe/relax_pfss_world.pdl]: I'm going to stop commenting on commented out code, but I think it'd be good practice to tidy up old code that's no longer being used. Maybe review all of your changed files here to scan for them and remove any that aren't needed in one single commit?

**Comment by GillySpace27** [comment]
Yeah, sounds good. Sorry, I’m always afraid to break things and not have the backup comments. But yeah, I understand that that’s git’s job and I should relax, hahaha.


From: Chris Lowder ***@***.***>
Date: Thursday, October 5, 2023 at 4:28 PM
To: lowderchris/fluxon-mhd ***@***.***>
Cc: Gilbert, Gilly ***@***.***>, Assign ***@***.***>
Subject: Re: [lowderchris/fluxon-mhd] Ingesting adapt (PR #38)
[EXTERNAL EMAIL]

@lowderchris commented on this pull request.

________________________________

In flux-pipe/relax_pfss_world.pdl<https://urldefense.com/v3/__https:/github.com/lowderchris/fluxon-mhd/pull/38*discussion_r1348066148__;Iw!!D-JDmu3Lc2wo0Jiybg!bJfMXMiXNWptsWjQ5UkC3sMbGEleDpVKj_KnhtVPt1eNNnoec28NYirmZhgp7ti_4V5YIS9Qs0aIfF9EKNaV$>:

> @@ -220,6 +220,9 @@ sub relax_pfss_world {

         # my $short_world_out_path = shorten_path($world_out_dir);



         # # print "\n\tWorld saved to $short_world_out_path";

+    # print $this_world_orig;

I'm going to stop commenting on commented out code, but I think it'd be good practice to tidy up old code that's no longer being used. Maybe review all of your changed files here to scan for them and remove any that aren't needed in one single commit?

File: pdl/t/00-test.t
Comment (GillySpace27) [reply pdl/t/00-test.t:1]: ok, removed

**Review by GillySpace27 — COMMENTED** [review]
linted everywhere but pipe_helper.py

**Comment by GillySpace27** [comment]
I'm going to not touch this pull request again until we talk

**Comment by GillySpace27** [comment]
Ahh sorry, this time I really mean it. Everything works and it should even work on other people's computers.
